include/ibm/flight_recorder.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+#define FLIGHT_RECORDER_MAX_ENTRIES 300
```

Comment:
Just do this like we do other limits 
https://github.com/ibm-openbmc/bmcweb/blob/ac4fd01279f5f8873cca21c3e16c16da5d7d2cd6/http/http_connection.hpp#L53